main/src/ca/uwaterloo/flix/language/dbg/FormatJvmMethod.scala

```diff
@@ -0,0 +1,15 @@
+package ca.uwaterloo.flix.language.dbg
```

Comment:
Add a license to the top of this (copy-paste from any other file and change the name and date at the top)

Reply:
Done, but I left it out because none of the other files here have a license (I thought that there might be some reason why they weren't used here?)

Reply:
Because someone forgot :)